### DISCUSSION TITLE
OSC相关

### DISCUSSION THREAD

**Discussion by NINGS203** [diana7127/mpv.net-DW#42] [Q&A] [answered]
视频文件文件名很长，看进度条时影响观看体验，请问修改哪个文件可以让osc不显示视频文件名。
![屏幕截图 2024-08-27 220604](https://github.com/user-attachments/assets/f36f5f4a-ae75-47d1-b6c8-d2cd117cb601)

**Comment by gdm257** [comment, marked as answer]
`script-opts/` 文件夹下创建一个名为 `modernx.conf` 的文本文件

```
showtitle=no
```

**Reply by NINGS203** [reply]
谢谢指导，操作成功了。
